NOTICE.txt

```diff
@@ -1955,6 +1942,19 @@ For more information, please see http://www.bis.doc.gov
 
 See also http://www.apache.org/dev/crypto.html and/or seek legal counsel.
 
+--------------------------------------------------------------------
+Dependency: github.com/ph/goformation
```

Comment:
So this change saves us 5 characters? Is that the only difference?

Reply:
it's a bit more than that, It merges the cloudformation type into a single file, saving 71 chars on worst case.